operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -3253,6 +3270,226 @@ gr-metricname-cache:
     annotations: {}
     labels: {}
 
+datadog:
```

Comment:
Is this mandatory? Surprised to not see `enabled` flag

Reply:
Also some docs on what this is and how to use might be handy. Like a link to our docs